lang/test/org/partiql/lang/syntax/SqlParserTestBase.kt

```diff
@@ -150,75 +144,42 @@ abstract class SqlParserTestBase : TestBase() {
-        assertEquals(actualStatement, transformedActualStatement)
+        // Check equal after transformation: PIG AST -> SexpElement -> PIG AST
+        assertRoundTripPigAstToSexpElement(actualStatement)
 
-        // Convert from the PIG instance back to ExprNode and assert the result is the same as actualExprNode.
-        val transformedActualExprNode = MetaStrippingRewriter.stripMetas(transformedActualStatement.toExprNode(ion))
-        assertEquals(MetaStrippingRewriter.stripMetas(actualExprNode), transformedActualExprNode)
-    }
-
-    private fun assertRoundTripIonElementToPartiQlAst(actualElement: SexpElement, expectedElement: SexpElement) {
-        // #1 We can transform the actual PartiqlAst element.
+        // Check equal for actual and expected in transformed astStatement: astStatment -> SexpElement -> astStatement
         val transformedActualStatement = PartiqlAst.transform(actualElement)
-
-        // #2 We can transform the expected PartiqlAst element.
         val transformedExpectedStatement = PartiqlAst.transform(expectedElement)
-
-        // #3 The results of both transformations match.
         assertEquals(transformedExpectedStatement, transformedActualStatement)
 
-        // #4 Re-transforming the actual PartiqlAst element and check if it matches the expected AST.
+        // Check round trip for actual: SexpElement -> astStatement -> SexpElement
         val reserializedActualElement = transformedActualStatement.toIonElement()
         assertEquals(expectedElement, reserializedActualElement)
-
-        // #5 Re-serializing the expected PartiqlAst element matches the expected AST.
-        // Note:  because of #3 above, no need for #5.
     }
 
     /**
-     * Strips metas from the [actualExprNode] so they are not included in equivalence checks
-     * and round-trip the resulting [actualExprNode] AST through [toAstStatement] and [toExprNode].
-     *
-     * Verify that the result matches the original without metas.
+     * Check equal after transformation: PIG AST -> SexpElement -> PIG AST
      */
-    private fun assertRoundTripPigAstToExprNode(actualExprNode: ExprNode) {
-        val actualExprNodeNoMetas = MetaStrippingRewriter.stripMetas(actualExprNode)
-        val actualStatement = actualExprNodeNoMetas.toAstStatement()
-        val roundTripActualExprNode = actualStatement.toExprNode(ion)
+    private fun assertRoundTripPigAstToSexpElement(actualStatement: PartiqlAst.Statement) =
+        assertEquals(actualStatement, PartiqlAst.transform(actualStatement.toIonElement()) as PartiqlAst.Statement)
 
-        assertEquals(actualExprNodeNoMetas, roundTripActualExprNode)
-    }
+    /**
+     * Check equal after transformation: PIG AST -> ExprNode -> PIG AST
+     */
+    private fun assertRoundTripPigAstToExprNode(actualStatement: PartiqlAst.Statement) =
+        assertEquals(actualStatement, actualStatement.toExprNode(ion).toAstStatement())
```

Comment:
Change round trip from (`ExprNode` -> PIG AST -> `ExprNode`) into (PIG AST -> `ExprNode` -> PIG AST), since we are deprecating `ExprNode`.